.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "cSpell.words": [
+    "Abendshien",
+    "qtbot",
+    "Voxel"
+  ]
+}
```

Comment:
I'd say we want to have whole `.vscode` in `.gitignore`

Reply:
.vscode is now ignored